docs/management/api/upload-api.asciidoc

```diff
@@ -0,0 +1,100 @@
+[[upload-api]]
+=== Upload file to host
+
+Upload a file to the host running {elastic-defend}.
+
+You must have the `File Operations` Security Kibana privilege as part of your role and at least an Enterprise license to perform this action.
+
+==== Request URL
+
+`POST <kibana host>:<port>/api/endpoint/action/upload`
+
+The request must include:
+
+* the `Content-Type: multipart/form-data` HTTP header.
```

Comment:
If there's only one item in the list, this could be a single sentence:
```suggestion
The request must include the `Content-Type: multipart/form-data` HTTP header.
```